### ISSUE TITLE
Problem with ZAP-CLI

### ISSUE THREAD

**Issue by John33000** [Grunny/zap-cli#74]
I lunch the same command like you : zap-cli quick-scan --self-contained --spider -r -s xss http://....

I defined environment variables ZAP_PATH and ZAP_PORT. But I have error python like this : 
![image](https://user-images.githubusercontent.com/46779058/54741383-952af280-4bbe-11e9-9f89-ae129468cf35.png)

I scan VM with DVWA

**Comment by wheelq** [comment]
most probably you did not define API key. Find out by checking the zap.log
helped me

**Comment by Grunny** [comment]
Yeah, as @wheelq said, that error is nearly always due to a missing API key. Make sure you have also set your API key in `ZAP_API_KEY`.

**Comment by ghost** [comment]
in ubuntu where to set ZAP_API_KEY,  @Grunny , I have not found any logs on running the same command in zap.log
